scss/detail.scss

```diff
@@ -46,3 +45,22 @@
   padding: 24px;
   width: 900px;
 }
+
+.cover-container {
+  position: relative;
+  margin-right: 26px;
+}
+
+.badge {
+
+  &::after {
```

Comment:
When you have no properties set to the element, you can just use `.badge::after`. But looking at the pseudo-element, maybe it should be a pseudo-element of the `.cover-container`?